wordpress-mobile/WordPress-Android
# Add support for gifs to local media picker

Fixes #8123

Replaces Volley with Glide in the local media picker. Adds support for animated gif images.

I've completely removed `ThumbnailLoader.java` as Glide has it's own loader `com.bumptech.glide.load.data.mediastore.ThumbFetcher` which basically does the same thing.

To test:
1. Tap on the Create Post button
2. Tap on the Add Media button
3. Make sure images/videos from the device are correctly loaded and gif images are being animated

@mzorz Could you please review this PR, thanks!
